---
layout: default
title: Teschendorff Lab / software / FEM
categories:
  - software
permalink: /software/FEM/
---

<div class="bs-callout bs-callout-danger">
  <div class="row">
    <div class="col-lg-7" style="display: table-cell; vertical-align: middle;">
      <h3><b>FEM (Functional Epigenetic Modules) algorithm</b></h3>
      <br>
      <p>
        FEM can identify interactome hotspots of differential promoter methylation and differential expression,
        where an inverse association between promoter methylation and gene expression is assumed.
        A R package to identify interactome hotspots of differential promoter methylation and differential expression,
        where an inverse association between promoter methylation and gene expression is assumed.
      </p>
      <br>
    </div>

    <div class="col-lg-5" style="font-weight: bold;">
      <img src="{{ '/images/FEM/pic.png' | relative_url }}" alt="FEM" height="250px">
    </div>
  </div>
</div>

<br>

<div class="panel panel-default">
  <div class="panel-heading" role="tab" id="headingFEM">
    <h4 class="panel-title">
      <a data-toggle="collapse" data-parent="#accordion" href="#collapseFEM"
         aria-expanded="true" aria-controls="collapseFEM" style="color:#333">
        Download and Install
      </a>
    </h4>
  </div>

  <div id="collapseFEM" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingFEM">
    <div class="panel-body">

      <p><strong><a href="http://www.bioconductor.org/packages/release/bioc/html/FEM.html">[Bioconductor Link]</a></strong></p>

      <p><strong>Citation</strong></p>
      <p>
		- Jiao Y, Widschwendter M, Teschendorff AE. A systems-level integrative framework for genome-wide DNA methylation and gene expression data identifies differential gene expression modules under epigenetic control. <i>Bioinformatics</i> 2014 Aug 15;30(16):2360-6. 
      </p>
      <p>
        - Jones A, Teschendorff AE, Li Q, Hayward JD, Kannan A, et al. (2013) Role of DNA methylation and epigenetic
        silencing of HAND2 in endometrial cancer development. <i>PLoS Med</i> 10:e1001551.
      </p>

      <p><strong>Installation</strong></p>
      <p>To install this package, start R and enter:</p>
      <pre><code>source("http://bioconductor.org/biocLite.R")
biocLite("FEM")</code></pre>

      <p><strong>Documentation</strong></p>
      <p>To view documentation for the version of this package installed in your system, start R and enter:</p>
      <pre><code>browseVignettes("FEM")</code></pre>

      <p><strong><a href="{{ '/images/FEM/intro.pdf' | relative_url }}">PDF</a></strong>:
        A R package to identify interactome hotspots of differential promoter methylation and differential expression,
        where an inverse association between promoter methylation and gene expression is assumed.
      </p>
      <p><strong><a href="{{ '/images/FEM/manual.pdf' | relative_url }}">PDF</a></strong>: Reference Manual</p>

    </div>
  </div>
</div>

<div class="bigspacer"></div>
<div class="bigspacer"></div>
