---
layout: default
title: Teschendorff Lab / software / Signalling Entropy
categories:
  - software
permalink: /software/SignalEntropy/
---

<div class="bs-callout bs-callout-danger">
  <div class="row">
    <div class="col-lg-7" style="display: table-cell; vertical-align: middle;">
      <h3><b>Signalling Entropy</b></h3>
      <br>
      <p>
        This software implements a sample-level signalling entropy framework. Gene expression data should be
        positively valued (e.g. Affy/Illumina intensities or RNA-Seq count data). The mass action principle is used
        to construct the stochastic matrix over a protein interaction network.
      </p>
      <br>
    </div>

    <div class="col-lg-5" style="font-weight: bold;">
      <img src="{{ '/images/SignalEntropy/pic.png' | relative_url }}" alt="Signalling Entropy" height="250px">
    </div>
  </div>
</div>

<br>

<div class="panel panel-default">
  <div class="panel-heading" role="tab" id="headingSignalEntropy">
    <h4 class="panel-title">
      <a data-toggle="collapse" data-parent="#accordion" href="#collapseSignalEntropy"
         aria-expanded="true" aria-controls="collapseSignalEntropy" style="color:#333">
        Download and Install
      </a>
    </h4>
  </div>

  <div id="collapseSignalEntropy" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingSignalEntropy">
    <div class="panel-body">

      <p><strong><a href="http://sourceforge.net/projects/signalentropy/files/?source=navbar">[SourceForge Link]</a></strong></p>

      <p><strong>Citation</strong></p>
      <p>
		  - Teschendorff AE, Sollich P, Kuehn R. Signalling entropy: A novel network-theoretical framework for systems analysis and interpretation of functional omic data. <i>Methods</i> 2014 Jun 1;67(3):282-93.
      </p>

      <p><strong>Description of files</strong></p>

      <p><strong>1) dataSCM2.Rd:</strong> R data object containing gene expression matrix <code>avdataSCM2.m</code> (≈17,000 genes × 191 samples) from the Stem Cell Matrix 2 compendium. Cell type is in <code>phenoSCM2.lv[[1]]</code> (hESC=1, iPSC=2, differentiated=3). Row names are Entrez Gene IDs.</p>

      <p><strong>2) hprdAsigH-13Jun12.Rd:</strong> R data object containing a PPI network adjacency matrix <code>hprdAsigH.m</code>. Row names are Entrez Gene IDs.</p>

      <p><strong>3) sr.Rd:</strong> Auxiliary R object needed to run the vignette.</p>

      <p><strong>4) DoIntegPIN.R:</strong> R function to integrate a PPI network with a gene expression matrix and extract the maximally connected subnetwork.</p>

      <p><strong>5) CompSR.R:</strong> R function to compute the entropy rate (plus auxiliary functions).</p>

      <p><strong>6) CompSRana.R:</strong> R function to compute the entropy rate analytically assuming <em>detailed balance</em> (faster computation).</p>

      <p><strong>7) <a href="{{ '/images/SignalEntropy/intro.pdf' | relative_url }}">PDF</a></strong>: Vignette / manual.</p>

    </div>
  </div>
</div>

<div class="bigspacer"></div>
<div class="bigspacer"></div>